configs/retinanet/retinanet_r18_fpn_1x_coco.py

```diff
@@ -0,0 +1,12 @@
+_base_ = [
```

Comment:
Need to update both README and metafile.

Reply:
> Need to update both README and metafile.

Okay ~